Core/HLE/sceAtrac.cpp

```diff
@@ -577,9 +577,9 @@ int Atrac::Analyze() {
 	int loopFirstSampleOffset = 0;
 	firstSampleoffset = 0;
 
-	while (Memory::Read_U32(first.addr + offset) != RIFF_WAVE_MAGIC) {
+	while (Memory::ReadUnchecked_U32(first.addr + offset) != RIFF_WAVE_MAGIC) {
```

Comment:
Is the entire range checked to be safe?  Also these changes seem unrelated.

I know there were games that crashed before when this went too far and tried to scan wrong memory.

-[Unknown]